kuma/javascript/src/utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Helper function to get cookie value given a cookie name
+ * @params {string} cookie name
+ * @returns {string} cookie value
+ */
+export function getCookie(name) {
+    var cookieValue = null;
+    if (document.cookie && document.cookie !== '') {
```

Comment:
Nit (and I realize I didn't catch this in https://github.com/mozilla/kuma/pull/5469): Since an empty string is falsy, I think this can be simplified to `if (document.cookie) {`